addons/mail/models/mail_channel.py

```diff
@@ -791,6 +791,10 @@ def channel_info(self):
                     'fetched_message_id': cp.fetched_message_id.id,
                     'seen_message_id': cp.seen_message_id.id,
                 } for cp in members_by_channel[channel] if cp.partner_id], key=lambda p: p['partner_id'])
+                info['guestMembers'] = [('insert', sorted(list([{
```

Comment:
`[...for]` should already be creating a list, the extra `list()` looks uncessary